src/platform/user/profile/vet360/components/AddressField/AddressField.jsx

```diff
@@ -164,6 +165,7 @@ export default class AddressField extends React.Component {
         deleteDisabled={this.props.deleteDisabled}
         Content={AddressView}
         EditModal={AddressEditModal}
+        ValidationModal={AddressValidationModal}
```

Comment:
Seems like the ValidationModal could and should be treated like the EditModal. I would love to have the address validation screen as simply a different render state in the EditModal so we aren't popping modals open and closed to create the illusion of them being the same modal. Maybe we can try that later if JAWS still hates this.